src/IcecastMetadataParser.js

```diff
@@ -144,12 +147,12 @@ class IcecastMetadataReader {
 
   _enqueueMetadata(meta) {
     this._metadataQueue.push(meta);
-    if (this._onMetadataQueue) this._onMetadataQueue(meta.metadata);
+    if (this._onMetadataQueue)
+      this._onMetadataQueue({ metadata: meta.metadata, time: meta.time });
   }
 
   _dequeueMetadata() {
     const meta = this._metadataQueue.shift();
-    if (this._onMetadataQueue) this._onMetadataQueue(meta.metadata);
```

Comment:
This didn't make sense to call here because it is not a queue event, so it is being removed.